website_sale_custom_filter/models/website_sale_custom_filter_value.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Camptocamp
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import api, fields, models
+
+
+class WebsiteSaleCustomFilterValue(models.Model):
+    _name = "website.sale.custom.filter.value"
+    _description = "website.sale.custom.filter.value"
+
+    @api.depends("value_filter_id.domain")
+    def _compute_selected_product_templates(self):
+        ProductTemplate = self.env["product.template"]
+        for res in self:
+            res.selected_product_tmpl_ids = False
+            if res.value_filter_id and res.value_filter_id.domain:
+                res.selected_product_tmpl_ids = ProductTemplate.search(
+                    res.value_filter_id._get_eval_domain()
+                )
+
+    name = fields.Char(required=True, string="Value name")
+    sequence = fields.Integer(required=True, default=10)
+    custom_filter_id = fields.Many2one(
+        "website.sale.custom.filter",
+        ondelete="cascade",
+        required=True,
+        string="Filter ID",
+    )
+    # display = fields.Selection(
+    #     [("checkbox", "Checkbox")],
+    #     default="checkbox",
+    #     string="Display type",
+    # )
```

Comment:
note: this is because we decided we need to separate color values from checkbox values and treat them as separate filter type, not filter values. So we don't need to distinguish between display type anymore as it can be only one type and there is no choice.

Reply:
to trash then?